slides.md

```diff
@@ -4,7 +4,138 @@
 
 ---
 # The tools
+
 ---
+# JavaScript basics
+
+To declare a variable, you can either use `const` (for constants), or `let` (for variables that can change over time). Do not use `var`.
+
+```javascript
+const count = 10;
+
+for (let i = 0; i < count; i++) {
+  console.log(`Hello ${count}`);
+}
+```
+
+You can use single, double, and inverse quotes to create string literals, only inverse quotes can interpolate variables though.
+
+```javascript
+const name = "Daniele";
+
+console.log("Hello " + name);
+console.log('Hello ' + name);
+console.log(`Hello ${name}`);
+```
+
+To create arrays, you can use square brackets `[]`.
+
+```javascript
+const anArray = [1, 2, 3];
+```
+
+To create objects (string indexed dictionaries), you can use curly brackets.
+```javascript
+const obj = {
+  a: 1,
+  b: "Hello"
+  c: {
+    nested: true
+  }
+};
+```
+
+To create a function, you can use both the `function` keyword, or use the arrow `=>` syntax. Arrow functions can directly return a value as an expression, and don't need `()` around a single argument. They need `()` around the return value if they return an object.
+
+```javascript
+function say(name) {
+  console.log(`Hello ${name}`);
+};
+
+const say = (name) => {
+  console.log(`Hello ${name}`);
+};
+
+const double = n => n * 2;
+const makeConfig = () => ({debug: true, log: false}); // returns the object
+```
+
+You can _spread_ objects and arrays, which applies their content to the current expression. The order in which you spread your object **matters**.
```

Comment:
"...to the current expression parameters"?

Reply:
Maybe instead of that "to decide which ones overrides in case of objects with the same fields, earlier declarations are overridden"?

Reply:
550d0dc